ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly.yaml

```diff
@@ -83,6 +83,31 @@ resources:
       cpu: "1"
       memory: 1Gi
 tests:
+- as: aws-ipi-f9-longduration-apiserver
+  cron: 4 14 9,21,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      OCP_ARCH: arm64
+      TEST_SCENARIOS: API_Server
+      TEST_TIMEOUT: "600"
+    test:
+    - ref: openshift-extended-test-longduration
+    workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f9-longduration-hive-remaining
```

Comment:
If I remember right, hive test cases can not executed in a arm cluster, cc @jianping-shu to confirm.

Reply:
You are right! Hive operator can be installed on amd cluster only so far.